src/client/pages/OfferNew/Checkout/Sign.tsx

```diff
@@ -1,69 +1,33 @@
+import React from 'react'
 import styled from '@emotion/styled'
-import { colorsV2 } from '@hedviginsurance/brand/dist'
+import { colorsV3 } from '@hedviginsurance/brand/dist'
 import { motion } from 'framer-motion'
-import React from 'react'
-import ReactMarkdown from 'react-markdown/with-html'
 import { useMediaQuery } from 'react-responsive'
 import { Button } from 'components/buttons'
 import { Spinner } from 'components/utils'
-import {
-  InsuranceTermType,
-  SignStatus as GraphQLSignStatus,
-} from 'data/graphql'
-import { OfferData } from 'pages/OfferNew/types'
-import { isNorwegian, isSwedish } from 'pages/OfferNew/utils'
+import { SignStatus as GraphQLSignStatus } from 'data/graphql'
 import { useTextKeys } from 'utils/textKeys'
 import { SignStatus } from './SignStatus'
 
-export const SignSpacer = styled('div')`
-  height: 250px;
-`
-const Wrapper = styled('div')`
-  position: absolute;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  width: 100%;
-  padding: 0 8rem 2.5rem 4.5rem;
-  background-image: linear-gradient(
-    to bottom,
-    rgba(249, 250, 252, 0),
-    ${colorsV2.offwhite} 50%
-  );
-
-  @media (max-width: 40rem) {
-    padding: 1rem;
-    padding-top: 0;
-  }
-`
+type WrapperProps = {
+  isDesktop: boolean
+  maxWidth?: number
+}
 
-const ButtonWrapper = styled('div')`
-  display: flex;
+const Wrapper = styled('div')<WrapperProps>`
+  background: ${colorsV3.gray100};
+  box-shadow: 0 -1px 4px ${colorsV3.gray500};
   width: 100%;
-
-  @media (max-width: 600px) {
-    justify-content: center;
-  }
+  max-width: ${({ maxWidth }) => (maxWidth ? `${maxWidth}px` : '100%')};
+  padding: ${({ isDesktop }) => (isDesktop ? '1.5rem 2rem' : '1rem')};
```

Comment:
Maybe it's just me, but can't we just use a good ol' media query instead of sending a js calculated value. *old man shakes fist at computer 😆

Reply:
Haha! 😍 No but my idea was that we'll _centralise_ the media queries later on so we always use the same values everywhere. Like we'll have a nice li'l hook that returns `isDesktop` for example. I guess I made it this way to make it easier to change that. And I actually like a named variable more than a regular media query - to me it's clearer what the media query is for then.

Reply:
I agree that is nice to have one value as reference. :) In this case we could do like we do in a few other places, set a const at the top of the file `const DESKTOP = 600` or and then we could use it both like so. Then we get the best of both worlds! 👯‍♂️
```
[`@media (min-width: ${DESKTOP}px)`]: {
...
}
....
const isDesktop = useMediaQuery({ minWidth: DESKTOP })
```

Reply:
And I think it can be hard to use the same `isDesktop` for all components since the component layout will be more contextual. But I we could definitely go over the breakpoints we have and se if we can find a pattern to standardise a few of them 🙌

Reply:
But maybe you just meant the `maxWidth`? In that case, see comments below (TLDR: I'll remove it).

Reply:
What I think could be a problem about useMediaqueries  is that you introduce styling into the component logic. I feel like you should not introduce it where it is maybe not needed (It absolutely has its place sometimes!).  I do however like that it becomes more declarative when the styles should be applied. 
In another project I think we went about it like this, declaring constants for MediaQueries pixelValues, but that might not really add anything to the readability more than just centralising pixelValues we use for mobile etc. 
`
...
@media (max-width: ${MOBILE_WIDTH}px){
`
This is more thoughts in general, I think the pr look super! 💪 


EDIT: Haha I am to slow @gustafguner ... 🐌

Reply:
No sure, you're right - this centralising can be done without using a hook, I was confusing things. I'm changing it to a media query (I still think we should use the same breakpoints everywhere, but this is something we should discuss later and not now).

Reply:
> EDIT: Haha I am to slow @gustafguner ... 🐌

@virrios  Wrong Gustav 😳  But I'm also too slow apparently haha! Anyways, YES valid points both of you!